bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -138,6 +138,12 @@ spec:
           preStop:
             exec:
               command: ["rm","-f","/etc/cni/net.d/80-openshift-network.conf", "/host/opt/cni/bin/openshift-sdn"]
+        readinessProbe:
+          exec:
+            # openshift-sdn writes this file when it is ready to handle pod requests.
+            command: ["ls", "/etc/cni/net.d/80-openshift-network.conf"]
```

Comment:
does kubelet print out stderr of exec probes? Anyway, wouldn't `test -f` be more idiomatic anyway?

Reply:
good point, `test -f` is better. No clue about exec probe logging.